### PR TITLE
Auth pages UI

### PR DESCRIPTION
**Pull request by sharathnair9999**
Create Signup, Login, and Forgot Password screens

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
everything is looking fine , try to put some error handling messages .

**Review by spraveenofficial — APPROVED** [review]

**Review by SukanyaSen530 — APPROVED** [review]

File: src/pages/Product/Product.js [hunk 1]
```diff
@@ -29,6 +29,17 @@ const Product = () => {
   };
 
   useEffect(() => {
+
+    (async()=>{
+      try{
+        const res = await axios.post("/api/auth/login", {email:"adarshbalika@gmail.com", password:"adarshbalika"})
+        console.log("login",res);
```
Comment (Mohitkumar33) [inline src/pages/Product/Product.js:36]: you can remove console.log

**Review by Mohitkumar33 — APPROVED** [review]
code arranged nicely

File: src/pages/Product/Product.js (hunk 1 above)
Comment (sharathnair9999) [reply src/pages/Product/Product.js:36]: yeah that was for test purpose will do that

**Pull request merged**